create vancancies in edicts

Dudalara opened the pull request.

Dudalara committed:
create vancancies in edicts

Dudalara committed:
create vancancies in edicts

Dudalara committed:
create vancancies in edicts

Dudalara committed:
add env dusk example

MarczalTSIGP commented on .env.testing:
Please, not change this file. It is configured correct to docker compose services.

MarczalTSIGP commented on .env.dusk.local:
This file is your configuration of dusk, this not should be in the repository. We just have the example on .env.dusk.local.example.

MarczalTSIGP commented on env.dusk.local.example:
This file is in the repository, why added it again. Did you do the right branch, I mean from master updated?

dmarczal committed:
Remove .env.dusk.local

Each developer must setup yours .env.dusk.local

dmarczal committed:
Added .env.testing.example

- Removed .env.testing because sensitive data

MarczalTSIGP commented on database/migrations/2023_04_24_221230_create_edict_units_table.php:
```diff
@@ -0,0 +1,48 @@
+            ->onDelete('cascade');
+
+            $table->integer('available_vacancies');
+            $table->enum('type_of_vacancy', ['RELEASED','REGISTERED']);
```
Why these fields are here? It not looks from this task.

dmarczal committed:
Changes files to correct:

1. Editorconfig to 4 spaces in php files.
2. Remove credetials from .env.dusk.example.
3. Remove line not used in create edict units.
4. Correct indent to create vacanccies edicts test.
5. Correct run linters

MarczalTSIGP reviewed (changes requested):
1. Where a see the vacancies of an edict? I guess you can put the list of vacancies in the same page of the register. And it's necessary to allow to edit or remove the vacancies. See PDF screen.

2. Did you change the populate to consider the number of vacancies?

3. If I removed an edict it removed the vacancies, did you write a test for this?

dmarczal committed:
CI: Changes to fixed CI env for test

dmarczal committed:
CI: Create .env.testing in githubactions

dmarczal committed:
CI: Create fixed APP name

dmarczal committed:
CI: Create fixed dusk env
